python/lsst/pex/exceptions/wrappers.py

```diff
@@ -1,7 +1,10 @@
+from future import standard_library
+standard_library.install_aliases()
```

Comment:
That's the 64k question. Futurize (the script) knows when to do it based on the content of the file that is being futurized. See http://python-future.org/compatible_idioms.html for examples of how to work around portability issues. You do not want to do it in every file.